autoscoper/CMakeLists.txt

```
@@ -136,6 +136,11 @@ find_package(GLEW REQUIRED CONFIG)
 target_link_libraries(autoscoper PUBLIC GLEW::GLEW)
 target_compile_definitions(autoscoper PUBLIC -DUSE_GLEW)
 
+if(NOT Autoscoper_INSTALL_SAMPLE_DATA)
+  set(Autoscoper_DISABLE_SAMPLE_DATA 1)
+  target_compile_definitions(autoscoper PUBLIC Autoscoper_DISABLE_SAMPLE_DATA)
+endif()
```

To avoid introducing a new variable, I suggest to inverse the logic:

```suggestion
target_compile_definitions(autoscoper PRIVATE $<$<BOOL:${Autoscoper_INSTALL_SAMPLE_DATA}>:Autoscoper_INSTALL_SAMPLE_DATA>)
```

... to support this, you would have the menu entries disabled by default.